Tests/Functional/Asynchronous/Predicates/Expectations/main.swift

```diff
@@ -0,0 +1,83 @@
+// RUN: %{swiftc} %s -o %{built_tests_dir}/Asynchronous-Predicates
+// RUN: %{built_tests_dir}/Asynchronous-Predicates > %t || true
+// RUN: %{xctest_checker} %t %s
+
+#if os(Linux) || os(FreeBSD)
+    import XCTest
+    import Foundation
+#else
+    import SwiftXCTest
+    import SwiftFoundation
+#endif
+
+// CHECK: Test Suite 'All tests' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Suite '.*\.xctest' started at \d+:\d+:\d+\.\d+
+
+// CHECK: Test Suite 'PredicateExpectationsTestCase' started at \d+:\d+:\d+\.\d+
+class PredicateExpectationsTestCase: XCTestCase {
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyTruePredicateAndObject_passes' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyTruePredicateAndObject_passes' passed \(\d+\.\d+ seconds\).
+    func test_immediatelyTruePredicateAndObject_passes() {
+        let predicate = NSPredicate(value: true)
+        let object = NSObject()
+        expectation(forPredicate:predicate, evaluatedWithObject: object)
+        waitForExpectations(withTimeout: 0.1)
+    }
+
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyFalsePredicateAndObject_fails' started at \d+:\d+:\d+\.\d+
+// CHECK: .*/Tests/Functional/Asynchronous/Predicates/Expectations/main.swift:34: error: PredicateExpectationsTestCase.test_immediatelyFalsePredicateAndObject_fails : Asynchronous wait failed - Exceeded timeout of 0.1 seconds, with unfulfilled expectations: Expect `<NSPredicate: 0x[0-9A-Fa-f]{1,16}>` for object <NSObject: 0x[0-9A-Fa-f]{1,16}>
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyFalsePredicateAndObject_fails' failed \(\d+\.\d+ seconds\).
+    func test_immediatelyFalsePredicateAndObject_fails() {
+        let predicate = NSPredicate(value: false)
+        let object = NSObject()
+        expectation(forPredicate:predicate, evaluatedWithObject: object)
+        waitForExpectations(withTimeout: 0.1)
+    }
+
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_delayedTruePredicateAndObject_passes' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_delayedTruePredicateAndObject_passes' passed \(\d+\.\d+ seconds\).
+    func test_delayedTruePredicateAndObject_passes() {
+        let halfSecLaterDate = NSDate(timeIntervalSinceNow: 2.0)
+        let predicate = NSPredicate(block: {
+            evaluatedObject, bindings in
+            if let evaluatedDate = evaluatedObject as? NSDate {
+                return evaluatedDate.compare(NSDate()) == NSComparisonResult.OrderedAscending
+            }
+            return false
+        })
+        expectation(forPredicate:predicate, evaluatedWithObject: halfSecLaterDate)
+        waitForExpectations(withTimeout: 3.0)
```

Comment:
A test that passes after two seconds is a little too slow for my tastes. Can we change these to `0.1` and `0.2` seconds instead?